Post.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  Post.swift
+//  A3
+//
+//  Created by Antoinette Marie Torres on 10/26/23.
+//
+
+import Foundation
```

Comment:
Um is this from the iOS course lol. make sure to delete this.

Reply:
HAHA omg how did this get here